lib/cylc/review.py

```diff
@@ -302,18 +314,27 @@ def taskjobs(
         else:
             page = 1
 
+        # Set list of task states depending on Cylc version 7 or 8
+        from sqlite3 import ProgrammingError
+        try:
+            if self.suite_dao.is_cylc8(user, suite):
+                task_statuses_ordered = CYLC8_TASK_STATUSES_ORDERED
+            else:
+                task_statuses_ordered = TASK_STATUSES_ORDERED
+        except ProgrammingError:
+            task_statuses_ordered = TASK_STATUSES_ORDERED
+
```

Comment:
Minor suggestion to replace two assignments with one.
```suggestion
        task_statuses_ordered = TASK_STATUSES_ORDERED
        try:
            if self.suite_dao.is_cylc8(user, suite):
                task_statuses_ordered = CYLC8_TASK_STATUSES_ORDERED
        except ProgrammingError:
            pass
```